kvbc/src/merkle_tree_storage_factory.cpp

```diff
@@ -40,8 +40,8 @@ std::shared_ptr<rocksdb::Statistics> completeRocksDBConfiguration(
 
   // Use the same block cache and table options for all column familes for now.
   for (auto& d : cf_descs) {
-    auto* cf_table_options = reinterpret_cast<::rocksdb::BlockBasedTableOptions*>(
-        d.options.table_factory->GetOptions<::rocksdb::BlockBasedTableOptions>());
+    auto* cf_table_options =
+        reinterpret_cast<::rocksdb::BlockBasedTableOptions*>(d.options.table_factory->GetOptions());
```

Comment:
is this change needed?

Reply:
it's a revert of a change needed for upgrading RocksDB